app/components/register.jsx

```diff
@@ -106,7 +106,7 @@ export class Register extends Component {
                   </h2>
                   <div className="form-group">
                     <label htmlFor="dex_title">Title</label>
```

Comment:
The username hasn't been finalized but we could use what they've already typed into the username input, right? And just default it to ashketchum10 if its empty, like what we do with Living Dex for title?